app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -62,7 +62,7 @@
                 %td
                   %span.js-test-case-number
                     = link_to claim.case_number , case_workers_claim_path(claim)
-                    %div.unique-id-small= claim.unique_id
+                    %span.unique-id-small= claim.unique_id
```

Comment:
Will have to look at it again. Div cannot be side a span. Will have to apply some display styling